continuous_integration/travis/run_tests.sh

```diff
@@ -13,3 +13,7 @@ else
     echo "py.test dask --runslow $XTRATESTARGS"
     py.test dask --runslow $XTRATESTARGS
 fi
+
+# This needs to be enabled to test __array_function__ protocol with
+# NumPy v1.16.x, enabled by default starting in v1.17
+export NUMPY_EXPERIMENTAL_ARRAY_FUNCTION=1
```

Comment:
Hmm, alright, sounds like that's a better place to have this, I'll move it there.

Reply:
Oh, but we don't have a build with NumPy v1.16.x, so there's nowhere to add the environment variable. It would be nice if after this commit we could start testing that, so:

1. Should I add a build for v.1.16.x (I suggest `x = 2`, the latests)?
2. Adding the build there is all we have to do, or is there some other configuration (and perhaps package installing) that needs to be done on Travis' side?

Reply:
I recommend bumping the NumPy version for the build that is currently at 1.15 to 1.16 and then adding this environment variable to that build as well.

You shouldn't have to do any other build or configuration.